modules/installation-aws-limits.adoc

```diff
@@ -53,7 +53,7 @@ link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-ei
 Review the
 link:https://aws.amazon.com/about-aws/global-infrastructure/[AWS region map] to
 determine how many availability zones are in each region. You can install a
-single cluster in many regions without increasing your EIP limit, but to take
+single cluster in most regions without increasing your EIP limit, but to take
```

Comment:
@bparry02, I'd rather say "many of these" than "most." (I don't track data on what percentage of regions you can install without increasing your EIP limit, so we might be introducing a different inaccuracy with "most.")